src/promptflow-tools/promptflow/tools/aoai_gpt4v.py

```diff
@@ -7,12 +7,130 @@
 from promptflow._internal import ToolProvider, tool
 from promptflow.connections import AzureOpenAIConnection
 from promptflow.contracts.types import PromptTemplate
+from typing import List, Dict
 
 from promptflow.tools.common import render_jinja_template, handle_openai_error, parse_chat, \
     preprocess_template_string, find_referenced_image_set, convert_to_chat_list, normalize_connection_config, \
     post_process_chat_api_response
 
 
+GPT4V_VERSION = "vision-preview"
+
+
+def _get_credential():
+    from azure.identity import DefaultAzureCredential
```

Comment:
what the behavior if import error occurs here?

Reply:
In runtime, promptflow  is  isntalled with promptflow[azureml-serving], it includes azure.identity and azure.ai.ml.
It will not fail here.
If there's import error, use DefaltCredential Instead.